OWNERS

```diff
@@ -6,3 +6,6 @@ approvers:
 - akvilemar
 reviewers:
 - j-fuentes
+- munnerz
```

Comment:
Can I request to be removed from the top-level reviewers list? I'm not actively working on the project so I'm unlikely to respond to reviews, and my inbox is already 🔥

Reply:
> unlikely to respond to reviews

unless explicitly requested of course :)